filebeat/module/traefik/_meta/kibana/6/dashboard/Filebeat-traefik-overview.json

```diff
@@ -2,597 +2,597 @@
     "objects": [
         {
             "attributes": {
-                "description": "", 
+                "description": "",
```

Comment:
Dashboards should not be updated yet. Can you rever this part?

Reply:
done